_includes/devdoc/guide_transactions.md

```diff
@@ -645,22 +645,16 @@ enable locktime.
 
 {% autocrossref %}
 
-Transactions typically pay transaction fees based on the total byte size
-of the signed transaction.  The transaction fee is given to the
+Transactions pay fees based on the total byte size of the signed transaction. Fees per byte are calculated based on current demand for space in mined blocks with fees rising as demand increases.  The transaction fee is given to the
 Bitcoin miner, as explained in the [block chain section][section block chain], and so it is
 ultimately up to each miner to choose the minimum transaction fee they
 will accept.
 
-<!-- TODO: check: 50 KB or 50 KiB?  Not that transactors care... -->
+There is also a concept of so-called "[high-priority transactions][/en/glossary/high-priority-transaction]{:#term-high-priority-transactions}{:.term}" which spend satoshis that have not moved for a long time.
 
-By default, miners reserve 50 KB of each block for [high-priority
-transactions][/en/glossary/high-priority-transaction]{:#term-high-priority-transactions}{:.term} which spend satoshis that haven't been spent for a long
-time.  The remaining space in each block is typically allocated to transactions
-based on their fee per byte, with higher-paying transactions being added
-in sequence until all of the available space is filled.
+In the past, these "priority" transaction were often exempt from the normal fee requirements. Before Bitcoin Core 0.12, 50 KB of each block would be reserved for these high-priority transactions, however this is now set to 0 KB by default.  After the priority area, all transactions are prioritized based on their fee per byte, with higher-paying transactions being added in sequence until all of the available space is filled. <!-- Consider adding links to blockmaxsize and blockmaxweight options once available in the glossary. -->
 
-As of Bitcoin Core 0.9, transactions which do not count as high-priority transactions
-need to pay a [minimum fee][/en/glossary/minimum-relay-fee]{:#term-minimum-fee}{:.term} (currently 1,000 satoshis) to be
+Bitcoin Core 0.9, added a [minimum fee][/en/glossary/minimum-relay-fee]{:#term-minimum-fee}{:.term} (currently 1,000 satoshis) to be
```

Comment:
Pretty sure the minimum fee was always there, but 0.9 just changed the logic around trickling below-min-fee somewhat. It's only being explicitly disabled (by default) in 0.14. Perhaps these details are irrelevant, though, since in practice a minimum fee is needed to broadcast.